index.rst

```diff
@@ -596,7 +730,7 @@ This work is being generalized to ComCam and the main camera, with the intention
 
 This work is being further tracked under `SITCOM-190 <https://jira.lsstcorp.org/browse/SITCOM-190>`_ which includes links to the detailed phased implementation plans and an evolving set of implementation JIRAs.
 
-We also recommend making use of existing URL interfaces to the RSP Portal image visualization tool, based on Firefly, so that, starting from the CVT, an individual detector image can be brought into Firefly for more detailed inspection. 
+We also recommend making use of existing URL interfaces to the RSP Portal image visualization tool, based on Firefly, so that, starting from the CVT, an individual detector image can be brought into Firefly for more detailed inspection.
```

Comment:
I'm not sure if we want to endorse firefly in this context.  E.g. something based on astrowidgets (== ginga) might meet our needs better -- of course, firefly might be the best solution.

Reply:
We certainly can generalize this... but I think this is outside the scope of this PR. 
I'm guessing Angelo ran something that removed an empty space at the end of the line which is why it got flagged as a change.